src/Symfony/Bridge/Twig/Extension/DumpExtension.php

```diff
@@ -67,11 +70,14 @@ public function dump(\Twig_Environment $env, $context)
         }
 
         $dump = fopen('php://memory', 'r+b');
-        $dumper = new HtmlDumper($dump);
+        $prevOutput = $this->dumper->setOutput($dump);
```

Comment:
What do you recommend? Should I require `AbstractDumper` in constructor? I don't think it would be ok to assume the dumper has null output.